packages/core/security/core-security-browser/src/api_provider.ts

```diff
@@ -14,7 +14,7 @@ import type { CoreAuthenticationService } from './authc';
  *
  * @public
  */
-export interface CoreSecurityContract {
+export interface CoreSecurityDelegateContract {
```

Comment:
Renamed for consistency with the userProfile naming - as I find something I think is better

Reply:
It's fine for now. We can come up with something 'better' when moving the implementation.